Application/EdFi.Ods.AdminApi.AdminConsole/Infrastructure/DataAccess/Models/Permission.cs

```diff
@@ -0,0 +1,15 @@
+// SPDX-License-Identifier: Apache-2.0
+// Licensed to the Ed-Fi Alliance under one or more agreements.
+// The Ed-Fi Alliance licenses this file to you under the Apache License, Version 2.0.
+// See the LICENSE and NOTICES files in the project root for more information.
+
+namespace EdFi.Ods.AdminApi.AdminConsole.Infrastructure.DataAccess.Models;
+
+public class Permission : IPermission
+{
+    public int? DocId { get; set; }
+    public required int PermissionId { get; set; }
```

Comment:
Could you clarify the reasoning behind adding the `PermissionId` field to the entity?